pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -90,24 +90,27 @@ type inflightMigrationAborted struct {
 	abortStatus v1.MigrationAbortStatus
 }
 
-func generateMigrationFlags(isBlockMigration, isUnsafeMigration, allowAutoConverge, allowPostyCopy, migratePaused bool) libvirt.DomainMigrateFlags {
+func generateMigrationFlags(isBlockMigration, migratePaused bool, options *cmdclient.MigrationOptions) libvirt.DomainMigrateFlags {
 	migrateFlags := libvirt.MIGRATE_LIVE | libvirt.MIGRATE_PEER2PEER | libvirt.MIGRATE_PERSIST_DEST
 
 	if isBlockMigration {
 		migrateFlags |= libvirt.MIGRATE_NON_SHARED_INC
 	}
-	if isUnsafeMigration {
+	if options.UnsafeMigration {
 		migrateFlags |= libvirt.MIGRATE_UNSAFE
 	}
-	if allowAutoConverge {
+	if options.AllowAutoConverge {
 		migrateFlags |= libvirt.MIGRATE_AUTO_CONVERGE
 	}
-	if allowPostyCopy {
+	if options.AllowPostCopy {
 		migrateFlags |= libvirt.MIGRATE_POSTCOPY
 	}
 	if migratePaused {
 		migrateFlags |= libvirt.MIGRATE_PAUSED
 	}
+	if options.ParallelMigrationThreads != nil {
```

Comment:
Are `0` and `1` valid values? E.g. what happens if `ParallelMigrationThreads` is set to `0`? Also, should we enable the parallel migration flag if `ParallelMigrationThreads` is `1`?

Reply:
Thanks a lot for bringing this up!
Can you PTAL on my last commit? It rejects Kubevirt CR / migration policies with `ParallelMigrationThreads <= 1`.

p.s. - For every migration config, we basically need to perform very similar check when it comes to Kubevirt CR and migration policies. As a follow-up PR, we should somehow converge this logic into one place so it will both be checked in the same way for the CR and the policies, and the code won't be duplicated.

Thanks a lot!